### PR TITLE
Lars example file for sprint 1.3

### PR DESCRIPTION
**Pull request by larswik**
Example file for sprint 1.3

## File

Example file for sprint 1.3 first draft.

## Covered Concept Templates - Usages Included

1. Project Unit Assignment (1.1)
2. Project Representation Context (1.1)
3. Project Declaration (1.1)
4. Project Classification Information (1.2)
5. Spatial Decomposition (1.2)
6. Project Global Positioning - B (1.2)
7. Project Document Information (1.3)
8. Project Dataset Information (1.3)
9. Project Library Information (1.3)
10. Project Type Definitions (1.3)
11. Project Template Definitions

## Miscellaneous

Created with GeometryGymIFC_Core (0.2.11)

### CODE REVIEW

**Comment by pjanck** [comment]
I'd wait until the checker is online to approve. Looking through the example manually, I can't seem to spot any mistakes.

**Comment by larswik** [comment]
> I'd wait until the checker is online to approve. Looking through the example manually, I can't seem to spot any mistakes.

Of course. Since I am off duty tomorrow, I just thought that I could have a file ready for thursday.

File: files/project_team/IFCTunnelDep1.3.ifc [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#2= IFCAXIS2PLACEMENT3D(#1,$,$);
+#3= IFCLOCALPLACEMENT($,#2);
+#4= IFCTUNNEL('0ZhDb1EcrFlB7Pt8ThZRKT',#10,'My Tunnel',$,$,#3,$,$,$,.ROAD.);
+#5= IFCPERSON('lars',$,$,$,$,$,$,$);
+#6= IFCORGANIZATION($,'Unknown',$,$,$);
+#7= IFCPERSONANDORGANIZATION(#5,#6,$);
+#8= IFCORGANIZATION($,'Geometry Gym Pty Ltd',$,$,$);
+#9= IFCAPPLICATION(#8,' v1.0.0.0','IFCTunnelDep1 v1.0.0.0','IFCTunnelDep1 v1.0.0.0');
+#10= IFCOWNERHISTORY(#7,#9,$,.ADDED.,1676974955,$,$,1676974955);
+#11= IFCPROJECT('27JcVZzyP2Lfos66GaVpVr',#10,'SPRINT 1.2 example with Project Global Positioning',$,$,$,$,(#25),#12);
```
Comment (pjanck) [inline files/project_team/IFCTunnelDep1.3.ifc:28]: You might want to rename the project :)

File: files/project_team/IFCTunnelDep1.3.ifc [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+#25= IFCGEOMETRICREPRESENTATIONCONTEXT('3D','Model',3,0.0000000254,#23,#26);
+#26= IFCDIRECTION((0.0,1.0));
+#27= IFCGEOGRAPHICCRS('EPSG:4326',$,'World Geodetic System 1984 ensemble',$,$,$);
+#28= IFCRIGIDOPERATION(#25,#27,IFCPLANEANGLEMEASURE(24.4769747),IFCPLANEANGLEMEASURE(61.9908225),0.0);
+#29= IFCCLASSIFICATION('Triona AB',$,$,'My classification',$,$,$);
+#30= IFCRELASSOCIATESCLASSIFICATION('3ddD2x5TvBIvBOSwJAT$a4',#10,$,$,(#11),#29);
+#31= IFCBUILTSYSTEM('3fug7PpajBPAqauiUTIxqk',#10,'A system',$,$,.TUNNEL_SUPPORT.,$);
+#32= IFCRELREFERENCEDINSPATIALSTRUCTURE('2X824mwzn24wNfF5xM5M$D',#10,$,$,(#31),#4);
+#33= IFCRELDECLARES('0XxzEMhRjEsx6ATpetrXW7',#10,$,$,#11,(#31,#43,#45));
+#34= IFCDOCUMENTINFORMATION('ISBN: 978-91-7725-946-6','Titel: Krav - VGU, V\X2\00E4\X0\gars och gators utformning',$,'http://trafikverket.diva-portal.org/smash/get/diva2:1621114/FULLTEXT02.pdf',$,$,$,$,$,$,$,$,'application/pdf',$,$,$,$);
```
Comment (pjanck) [inline files/project_team/IFCTunnelDep1.3.ifc:51]: `Editors` field is empty, and thus an actor declaration by the project is missing as well.

File: files/project_team/IFCTunnelDep1.3.ifc [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+#35= IFCRELASSOCIATESDOCUMENT('2D6TlgIn99LP7LllSlTHuo',#10,$,$,(#11),#34);
+#36= IFCDATASETINFORMATION('Dataset-xxx','Dataset-xxx-xml',$,'http://xxx.yyy.se/Datasets/Dataset-xxx.xml',$,$,$,$,$,$,$,$,'text/xml,$,$,$,$,'http://geosciml.org/geosciml/2.0/xsd/geosciml.xsd');
+#37= IFCRELASSOCIATESDATASET('0lM02zei1F0OCnzjuEMfHz',#10,$,$,(#11),#36);
+#38= IFCLIBRARYINFORMATION('My library','1.0',#41,'2023-02-21T11:22:35','http://xxx.yyy.se/Libraries/MyLib.ifc',$);
+#39= IFCPERSON($,'Wikstr\X2\00F6\X0\m','Lars',$,$,$,$,$);
+#40= IFCORGANIZATION($,'Triona AB',$,$,$);
+#41= IFCPERSONANDORGANIZATION(#39,#40,$);
+#42= IFCRELASSOCIATESLIBRARY('3BtsGx9$19ZOr4YDjnzJf1',#10,$,$,(#11),#38);
+#43= IFCARCHELEMENTTYPE('1DlE0cqGX8XQh$z_muWlk4',#10,'Arch-123',$,$,$,$,$,'MyArchType',.USERDEFINED.);
+#45= IFCPROPERTYSETTEMPLATE('3ruRe82Pv35A6pl1gt8lzj',#10,'Pset_GeoObsBoreholeCommon',$,.PSET_TYPEDRIVENOVERRIDE.,'IfcBorehole',(#46));
```
Comment (pjanck) [inline files/project_team/IFCTunnelDep1.3.ifc:61]:
```suggestion
#45= IFCPROPERTYSETTEMPLATE('3ruRe82Pv35A6pl1gt8lzj',#10,'Pset_GeoObsBoreholeConstruction',$,.PSET_TYPEDRIVENOVERRIDE.,'IfcBorehole',(#46));
```

**Review by pjanck — CHANGES_REQUESTED** [review]
I have a local checker instance up and running. I would like to hear your opinion on these errors reported by it before I go public.

File: files/project_team/IFCTunnelDep1.3.ifc (hunk 2 above)
Comment (larswik) [reply files/project_team/IFCTunnelDep1.3.ifc]: Added an Editor. I don't quite understand why the IfcRelDeclares check is reporting an error? Currently the following is declared:
```
#33= IFCRELDECLARES('0uB3HDsab5m8q9MfuymxaG',#10,$,$,#11,(#31,#36,#45,#47));
#31= IFCBUILTSYSTEM('2qhchTsgr1_gWIcwPXP_ih',#10,'A system',$,$,.TUNNEL_SUPPORT.,$);
#35= IFCORGANIZATION('http://www.trafikverket.se','Trafikverket',$,$,$);
#36= IFCACTOR('2FCX85XmbF5hu3VP7$qYVD',#10,$,$,$,#35);
#45= IFCARCHELEMENTTYPE('03tHuUU5rC8AvZRAL_MSnn',#10,'Arch-123',$,$,$,$,$,'MyArchType',.USERDEFINED.);
#47= IFCPROPERTYSETTEMPLATE('1HnsV$PyzCJRmVHrw7NENU',#10,'Pset_GeoObsBoreholeConstruction',$,.PSET_TYPEDRIVENOVERRIDE.,'IfcBorehole',(#48));
```

**Review by pjanck — APPROVED** [review]
LFTM. The check somehow picks another file to check, thus failing ...

File: files/project_team/IFCTunnelDep1.3.ifc (hunk 2 above)
Comment (pjanck) [reply files/project_team/IFCTunnelDep1.3.ifc]: Found the error - the checker was set up wrongly.

**Pull request merged**